app/src/main/java/com/example/benchproject/app/common/navigator/FragmentNavigator.kt

```diff
@@ -16,7 +17,9 @@ abstract class FragmentNavigator {
             this.fragment = null
     }
 
-    fun errorSnackBar(message: String) {
-        Snackbar.make(fragment!!.requireView(), message, Snackbar.LENGTH_SHORT).show()
+    fun errorSnackBar(message: String, lambda: () -> Unit) {
```

Comment:
What `lambda` name actually means? Try to be more specific, now it's unclear what and why it will be called

Reply:
I changed it on `onAction`